.github/workflows/test-operator-wandb.yaml

```diff
@@ -34,29 +34,27 @@ jobs:
         uses: jlandowner/helm-chartsnap-action@v1
         id: chartsnap
         with:
-          chart:  charts/operator-wandb/
+          chart: charts/operator-wandb/
           values: test-configs/operator-wandb/
 
   test:
     name: Test Chart
     strategy:
       fail-fast: false
       matrix:
-        k8s-version: [
-            "v1.32.2",
-            "v1.31.6",
-            "v1.30.10",
-          ]
-        configuration: [
+        k8s-version: ["v1.32.2", "v1.31.6", "v1.30.10", "v1.29.14"]
```

Comment:
This addition of `1.29.14` is not part of my PR. Is this intentional?

Reply:
I think this is an artifact of the merge that I missed.